lib/src/build_importer.dart

```diff
@@ -53,12 +54,26 @@ class BuildImporter extends sass.AsyncImporter {
   /// If neither exists, returns an empty list.
   Future<List<AssetId>> _tryImport(String import) async {
     final imports = <AssetId>[];
-    final partialId = new AssetId.resolve(
-        p.url.join(p.dirname(import), '_${p.basename(import)}'),
-        from: _buildStep.inputId);
+    AssetId partialId;
+    if (_includePath == null) {
+      partialId = new AssetId.resolve(
+          p.url.join(p.dirname(import), '_${p.basename(import)}'),
+          from: _buildStep.inputId);
+    } else {
+      partialId = new AssetId.resolve(p.url.join(
+          'asset:${_buildStep.inputId.package}/${_includePath}',
+          p.dirname(import),
+          '_${p.basename(import)}'));
+    }
+
     if (await _buildStep.canRead(partialId)) imports.add(partialId);
-    final importId = new AssetId.resolve(import, from: _buildStep.inputId);
+
+    final importId = _includePath == null
+        ? new AssetId.resolve(import, from: _buildStep.inputId)
+        : new AssetId.resolve(p.url.join(
+            'asset:${_buildStep.inputId.package}/${_includePath}', import));
```

Comment:
Do we need to omit the `from` parameter on these `resolve()` calls?

Reply:
We can omit it because I'm assuming include paths are not relative.  From should only be necessary for relative imports.

Reply:
I ran tests adding the from parameter and they still all pass. I'd have to look for an expect case where they would fail.